app/views/space_contacts/_new.html.erb

```diff
@@ -7,6 +7,7 @@
     <div class="modal modal--with-header-and-footer">
         <div class="modal-header">
           <button type="button"
+                  id="space_contacts_new"|
```

Comment:
This isn't the button for a new space contact, but for closing the modal. (The ID is also duplicated on line 31) 

Probably just an error? I guess this ID should be removed?

Reply:
(or renamed, if it's used in testing)